deepchem/molnet/load_function/molnet_loader.py

```diff
@@ -47,31 +47,15 @@ def get_directory_name(self) -> str:
 
 
 featurizers = {
-    'graphconv': dc.feat.ConvMolFeaturizer(),
-    'weave': dc.feat.WeaveFeaturizer(),
+    'coulombmatrix': lambda: dc.feat.CoulombMatrix(29),
```

Comment:
Would we have the rdkit import error here now that we no longer have the try/catch block?

Reply:
This change is incorrect.  QM7, QM8, and QM9 each require a different size Coulomb matrix.  That's why it can't be specified with a string.

The CoulombMatrix constructor throws an exception if RDKit isn't installed, but not for any good reason.  The constructor doesn't use anything from RDKit.  We could move that check into the `coulomb_matrix()` method, which is the only place it gets used.

Reply:
I fixed.